prediction_market_agent/agents/think_thoroughly_agent/think_thoroughly_agent.py

```diff
@@ -188,13 +188,14 @@ def get_required_conditions(self, question: str) -> Scenarios:
         create_required_conditions = Task(
             description=CREATE_REQUIRED_CONDITIONS_PROMPT,
             expected_output=LIST_OF_SCENARIOS_OUTPUT,
-            output_json=Scenarios,
+            output_pydantic=Scenarios,
```

Comment:
These `Scenarios` cases weren't erroring, but changing these too to be consistent.